docs/dev_cycle.part-02.rst

```diff
@@ -0,0 +1,174 @@
+    solc, the solidity compiler commandline interface
+    Version: 0.4.13+commit.0fb4cb1a.Linux.g++
+    
+In the project directory:
+
+.. code-block:: shell
+
+    $ populus compile
+    
+If you copy-pasted the ``Donator`` contract example, you will get:
+
+.. code-block:: shell
+
+  Traceback (most recent call last):
+    File "/usr/local/bin/populus", line 11, in <module>
+      sys.exit(main())
+    File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 722, in __call__
+      return self.main(*args, **kwargs)
+    File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 697, in main
+      rv = self.invoke(ctx)
+    File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1066, in invoke
+      return _process_result(sub_ctx.command.invoke(sub_ctx))
+    File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 895, in invoke
+      return ctx.invoke(self.callback, **ctx.params)
+    File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 535, in invoke
+      return callback(*args, **kwargs)
+    File "/usr/local/lib/python3.5/dist-packages/click/decorators.py", line 17, in new_func
+      return f(get_current_context(), *args, **kwargs)
+    File "/usr/local/lib/python3.5/dist-packages/populus/cli/compile_cmd.py", line 29, in compile_cmd
+      compile_project(project, watch)
+    File "/usr/local/lib/python3.5/dist-packages/populus/api/compile_contracts.py", line 18, in compile_project
+      _, compiled_contracts = compile_project_contracts(project)
+    File "/usr/local/lib/python3.5/dist-packages/populus/compilation/__init__.py", line 54, in compile_project_contracts
+      import_remappings=project.config.get('compilation.import_remappings'),
+    File "/usr/local/lib/python3.5/dist-packages/populus/compilation/backends/solc_auto.py", line 52, in get_compiled_contracts
+      return self.proxy_backend.get_compiled_contracts(*args, **kwargs)
+    File "/usr/local/lib/python3.5/dist-packages/populus/compilation/backends/solc_standard_json.py", line 131, in get_compiled_contracts
+      compilation_result = compile_standard(std_input, **command_line_options)
+    File "/usr/local/lib/python3.5/dist-packages/solc/main.py", line 184, in compile_standard
+      message=error_message,
+  solc.exceptions.SolcError: contracts/Donator.sol:21:5: DeclarationError: Identifier not found or not unique.
+  uin in_usd = msg.value * usd_rate;
+  ^-^
+  
+      > command: `solc --standard-json`
+      > return code: `0`
+      > stderr:
+      {"contracts":{},"errors":[{"component":"general","formattedMessage":"contracts/Donator.sol:21:5: DeclarationError: Identifier not found or not unique.\n    uin in_usd = msg.value * usd_rate;\n    ^-^\n","message":"Identifier not found or not unique.","severity":"error","type":"DeclarationError"}],"sources":{}}
+  
+      > stdout:
+
+What's that? actually it's not that bad. You can ignore the Python traceback, which is just the Populus call stack until the actual
```

Comment:
> You can ignore the Python traceback

Maybe this would be a good time to have `populus` wrap that and not dump the whole traceback when it's a `SolcError`?